README.md

```diff
@@ -89,6 +90,8 @@ attached to that root event. The default value for the name is `CodecCsv`.
 
 **publish-header** - set to publish decoded header. The default value is `false`.
 
+**trim-whitespace** - set to trim whitespace in header (when default-header isn't set) and cell. The default value is `true`.
```

Comment:
I believe we should all trim all values in the default header if it is set. Don't think that someone intentionally will add a header with leading or trailing whitespaces.

Reply:
I will add trim for `default-header`

Reply:
Also, add blank check for header values

Reply:
All of that have already done )